src/codeu/chat/codeU_db/CreateDatabase.java

```diff
@@ -0,0 +1,98 @@
+package codeu.chat.codeU_db;
+
+import java.sql.*;
+
+public class CreateDatabase
+{
+    public static void main( String args[] )
+    {
+        Connection c = null;
+        Statement stmt = null;
+
+        try {
+            Class.forName("org.sqlite.JDBC");
+            c = DriverManager.getConnection("jdbc:sqlite:./bin/codeu/chat/codeU_db/ChatDatabase.db");
```

Comment:
Lots of code seems to be using this logic. Perhaps it should be moved to a static method in a helper class so you only need to update it in one place.